packages/SwingSet/test/vat-admin/create-vat.test.js

```diff
@@ -415,25 +424,27 @@ test('createVat holds refcount', async t => {
   t.deepEqual(c.dump().reapQueue, []);
   // console.log(`---`);
 
-  // At this point we expected to see 5,5: v2-bootstrap, kpResolution pin,
-  // vat-vat-admin (because of the non-dropping bug), device-vat-admin
```

Comment:
descriptions change that wasn't included in a previous fix?

Reply:
yeah, I think that's right, I should have removed the bit about "non-dropping bug" in the previous PR that fixed that issue.